JavaScript/test/dateExtractor-english.test.js

```diff
@@ -0,0 +1,90 @@
+var describe = require('ava-spec').describe;
+var EnglishDateExtractorConfiguration = require('../compiled/dateTime/english/dateConfiguration').EnglishDateExtractorConfiguration;
+var BaseDateExtractor = require('../compiled/dateTime/baseDate').BaseDateExtractor;
+var Constants = require('../compiled/dateTime/constants').Constants;
+
+describe('Date Extractor', it => {
+    let extractor = new BaseDateExtractor(new EnglishDateExtractorConfiguration());
+
+    basicTest(it, extractor, "I'll go back on 15", 16, 2);
```

Comment:
We should also think of a way to put the unit test data for both C# and JS into data files, so unit tests don't need to be added to both and kept in sync manually.